docs/docs/pricing/README.md

```diff
@@ -142,15 +138,10 @@ Each execution of this source would result 1 credit because:
 
 This would result in 1 credit per execution.
 
-The initial free credit only applies to Pipedream Public Registry sources attached to at least one workflow.
+- The initial free credit only applies to Pipedream Public Registry sources attached to at least one workflow.
```

Comment:
Same feedback. Single bullet so combine with the sentence above.

Reply:
I noticed a couple of other sections with one bullet like this that you didn't touch. Would you mind just going through and fixing those to be consistent?